### PR TITLE
Prepare a vial firmware

### PR DESCRIPTION
**Pull request by shofel**
todo

Major:
- [x] fix the number of keys in the thumb clusters

Cosmetic:
- [x] remove qmk_firmware submodule
- [x] unmap QK_BOOT key
- [x] cleanup commit history
- [x] fixup width of the keys of the outer columns

Maybe:
- [ ] rotate thumb clusters

### CODE REVIEW

**Comment by shofel** [comment]
I made work 

image

image

image

image

**Comment by shofel** [comment]
it lacks of visual presicion though. Columns are not staggered, and thumb clusters are not rotated. I just translated the matrix [LAYOUT](https://github.com/vial-kb/vial-qmk/blob/12950db4d8ec1f294b1285e9b554a8fdc0a4bc6d/keyboards/handwired/dactyl_manuform/5x6/5x6.h)

**Comment by shofel** [comment]
Here is a more precisely visualised dactyl 5x6_5, for reference
image

File: keymap/vial.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+      [{"x": 2 }, "4,2", "4,3"],
+      [{"x": 4 }, "4,4", "4,5"],
+      [{"x": 6 }, "5,4", "5,5"],
+      [{"x": 6 }, "5,2", "5,3"],
+      
+      [{"x": 14, "y": -4 },
+                          "10,2", "10,3"],
+      [{"x": 12 }, "10,0", "10,1"],
+      [{"x": 10 }, "11,0", "11,0"],
```
Comment (shofel) [inline keymap/vial.json:27]:
```suggestion
      [{"x": 10 }, "11,0", "11,1"],
```

**Comment by He4eT** [comment]
Rebased and reordered in #3

**Pull request closed without merging**